.github/workflows/pull_requests.yaml

```diff
@@ -67,3 +67,10 @@ jobs:
         with:
           name: mssql-test-results
           path: ${{ env.TEST_OUTPUT_DIR }}
+
+  e2e:
+    needs: build
+    uses: ryanemerson/gingersnap-test/.github/workflows/e2e.yaml@test_dev
```

Comment:
@pminz This will be updated to https://github.com/gingersnap-project/e2e once it's ready.